src/_pytest/config/__init__.py

```diff
@@ -203,7 +206,10 @@ def get_plugin_manager():
     return get_config().pluginmanager
 
 
-def _prepareconfig(args=None, plugins=None):
+def _prepareconfig(
+    args: Optional[Union[List, Tuple]] = None,
+    plugins: Optional[Sequence[Union[str, object]]] = None,
```

Comment:
When I (tried to) [add types to pluggy](https://github.com/pytest-dev/pluggy/pull/225), I also typed plugins as `object`. This is the best type for completely opaque things.

I wonder if for clarity we might want to add an alias `Plugin = object` to pluggy and then use it, this why it'd be a bit clearer when we refer to plugins. But just a thought, not for this PR.

Reply:
> I wonder if for clarity we might want to add an alias `Plugin = object` to pluggy and then use it, this why it'd be a bit clearer when we refer to plugins. But just a thought, not for this PR.

Makes total sense, also for this PR already then, of course.

But we should add it here, given that pluggy is not typed?

Reply:
Yeah, adding here for now makes sense -- better than nothing.